### PR TITLE
Move the Pipelines WG PST PM meeting to PST AM

### PR DESCRIPTION
**Pull request by rimolive**
As discussed in the Pipelines WG meeting, we will be meeting every 2 weeks only on PST AM.

cc @kubeflow/pipelines

### CODE REVIEW

File: calendar/calendar.yaml [hunk 1]
```diff
@@ -509,4 +494,20 @@
...
+  video: https://meet.google.com/jkr-dupp-wwm
+  until: 07/03/2024
+  attendees:
+    - email: kubeflow-discuss@googlegroups.com
+    - email: kubeflow-pipelines@google.com
+  description:
+    - |
+      Notes & Agenda: http://bit.ly/kfp-meeting-notes
+      Join at: https://meet.google.com/jkr-dupp-wwm
+  organizer: chensun
```
Comment (zijianjoy) [inline calendar/calendar.yaml:513]: Can we remove this meeting to avoid duplication?

Comment (rimolive) [reply calendar/calendar.yaml:513]: This is part of instructions to remove recurring meetings. We cannot just remove it from the file, but move to the bottom of the file and set a `until` field with a date in the past. If we don't do that, the calendar entry will never be removed.

Comment (zijianjoy) [reply calendar/calendar.yaml:513]: I see, thank you for the reminder!

**Comment by zijianjoy** [comment]
/lgtm
/approve

Thank you for this PR!

**Pull request merged**